src/snowflake/snowpark/types.py

```diff
@@ -407,6 +407,7 @@ class ColumnIdentifier:
     """Represents a column identifier."""
 
     def __init__(self, normalized_name: str) -> None:
+        self.raw_name = normalized_name
```

Comment:
If this is internal usage only then I prefer marking them as private `_raw_name`